Week3/homework/step2-7.js

```diff
@@ -20,4 +20,15 @@ f2(y);
 
 console.log(y);
 
-// Add your explanation as a comment here
+// Add your explanation as a comment
+/* 
+Changing the value of a variable never changes the underlying primitive or object. 
+However, changing a property of an object referenced by a variable does change the underlying object.
+*/
+
+/* 
+in my terms:
+the variable == x == is defined with (const). when the value of such a variable is a num or str, so it doesn't accept any changes.
+the variable == y == has object as a value. when the value of a variable is an arr or an obj, 
+so it accepts changes accessing it through keys for obj and [i] for arr.
+*/
```

Comment:
I think you understand what the difference is, make sure you really know the difference between a copy by value and a copy by reference and which is used in which situation.